common/rtc_manager_rviz_plugin/src/rtc_manager_panel.cpp

```diff
@@ -0,0 +1,331 @@
+
+namespace rviz_plugins
+{
+inline std::string Bool2String(const bool var) { return var ? "True" : "False"; }
+using std::placeholders::_1;
+using std::placeholders::_2;
+Module getModuleType(const std::string & module_name)
+{
+  Module module;
+  if (module_name == "blind_spot") {
+    module.type = Module::BLIND_SPOT;
+  } else if (module_name == "crosswalk") {
+    module.type = Module::CROSSWALK;
+  } else if (module_name == "detection_area") {
+    module.type = Module::DETECTION_AREA;
+  } else if (module_name == "intersection") {
+    module.type = Module::INTERSECTION;
+  } else if (module_name == "no_stopping_area") {
+    module.type = Module::NO_STOPPING_AREA;
+  } else if (module_name == "occlusion_spot") {
+    module.type = Module::OCCLUSION_SPOT;
+  } else if (module_name == "stop_line") {
+    module.type = Module::NONE;
+  } else if (module_name == "traffic_light") {
+    module.type = Module::TRAFFIC_LIGHT;
+  } else if (module_name == "virtual_traffic_light") {
+    module.type = Module::TRAFFIC_LIGHT;
+  } else if (module_name == "lane_change_left") {
+    module.type = Module::LANE_CHANGE_LEFT;
+  } else if (module_name == "lane_change_right") {
+    module.type = Module::LANE_CHANGE_RIGHT;
+  } else if (module_name == "avoidance_left") {
+    module.type = Module::AVOIDANCE_LEFT;
+  } else if (module_name == "avoidance_right") {
+    module.type = Module::AVOIDANCE_RIGHT;
+  } else if (module_name == "pull_over") {
+    module.type = Module::PULL_OVER;
+  } else if (module_name == "pull_out") {
+    module.type = Module::PULL_OUT;
+  }
+  return module;
+}
+
+std::string getModuleName(const uint8_t module_type)
+{
+  switch (module_type) {
+    case Module::LANE_CHANGE_LEFT: {
+      return "lane_change_left";
+    }
+    case Module::LANE_CHANGE_RIGHT: {
+      return "lane_change_right";
+    }
+    case Module::AVOIDANCE_LEFT: {
+      return "avoidance_left";
+    }
+    case Module::AVOIDANCE_RIGHT: {
+      return "avoidance_right";
+    }
+    case Module::PULL_OVER: {
+      return "pull_over";
+    }
+    case Module::PULL_OUT: {
+      return "pull_out";
+    }
+    case Module::TRAFFIC_LIGHT: {
+      return "traffic_light";
+    }
+    case Module::INTERSECTION: {
+      return "intersection";
+    }
+    case Module::CROSSWALK: {
+      return "crosswalk";
+    }
+    case Module::BLIND_SPOT: {
+      return "blind_spot";
+    }
+    case Module::DETECTION_AREA: {
+      return "detection_area";
+    }
+    case Module::NO_STOPPING_AREA: {
+      return "no_stopping_area";
+    }
+    case Module::OCCLUSION_SPOT: {
+      return "occlusion_spot";
+    }
+  }
+  return "NONE";
+}
+
+RTCManagerPanel::RTCManagerPanel(QWidget * parent) : rviz_common::Panel(parent)
+{
+  // TODO(tanaka): replace this magic number to Module::SIZE
+  const size_t module_size = 14;
+  auto_modes_.reserve(module_size);
+  auto * v_layout = new QVBoxLayout;
+  auto * auto_mode_table_layout = new QHBoxLayout;
```

Comment:
What is this layout for? I don't think it affects the layout as there is only one widget. The `auto_mode_table_` can be added directly to the `v_layout`.